templates/home.html

```diff
@@ -183,6 +253,38 @@ <h5 class="modal-title">Edit Animal</h5>
     // Initial fetch
     fetchAnimals();
 
+    document.getElementById('animal_type').addEventListener('change', function() {
+        var breedSelect = document.getElementById('animal_breed');
+        breedSelect.innerHTML = '';
+
+        var selectedType = this.value;
+
+        if (selectedType === 'cat') {
+            var catBreeds = ['Ragdoll', 'Sphynx cat', 'Bengal cat', 'Maine Coon', 'American Shorthair', 'Siamese cat',
+            'Persian cat', 'Abyssinian', 'Devon Rex', 'I dunno, it is just a lovely cat'];
+            catBreeds.forEach(function(breed) {
+                var option = document.createElement('option');
+                option.value = breed;
+                option.text = breed;
+                breedSelect.add(option);
+            });
+        } else if (selectedType === 'dog') {
+            var dogBreeds = ['Labrador Retriever', 'Bulldog', 'Poodle', 'Golden Retriever', 'German Shepherd',
+                'Rottweiler', 'French Bulldog', 'Beagle', 'Beagle', 'I dunno, it is just a lovely dog'];
+            dogBreeds.forEach(function(breed) {
+                var option = document.createElement('option');
+                option.value = breed;
+                option.text = breed;
+                breedSelect.add(option);
+            });
+        }
+    });
+
+    document.addEventListener('DOMContentLoaded', function() {
+    var today = new Date().toISOString().split('T')[0];
+    document.getElementById('birth_date').setAttribute('max', today);
```

Comment:
Maybe it's possible to do the same with jQuery instead of vanilla JS?

Reply:
@marcustas changed